types/node/buffer.d.ts

```diff
@@ -2,6 +2,11 @@ declare module "buffer" {
     export const INSPECT_MAX_BYTES: number;
     const BuffType: typeof Buffer;
 
+    export function transcode(
+        source: Buffer | Uint8Array,
+        fromEnc: "ascii" | "utf8" | "utf16le" | "ucs2" | "latin1" | "binary",
```

Comment:
Actually, please use `BufferEncoding` instead.

Reply:
I don't agree here as the values supported in transcode are less then usual.

Reply:
Ah, my bad, still worth creating a type to avoid duplication 😄

Reply:
Maybe `BufferTranscodeEncoding` or `TranscodeEncoding`?

Reply:
If it is inside the `buffer` module I would prefer `TranscodeEncoding` to avoid something like `buffer.BufferTranscodeEncoding`.

Reply:
Added as `TranscodeEncoding` inside `buffer` module.